myEnigma/myEnigma.ino

```diff
@@ -1431,7 +1437,7 @@ void parsePlugboard(char* plugboard){
     ch=plugboard[0];
     len=strlen(plugboard);
 
-    memmove(plugboard,plugboard+1,len);
+    plugboard++;
```

Comment:
This change would break the plugboard wipe. memmove is used to zero all plugboard settings, not to increase the plugboard value with one.